docs/codeql/codeql-language-guides/basic-query-for-cpp-code.rst

```diff
@@ -134,14 +141,38 @@ To exclude ``if`` statements that have an ``else`` branch:
         block.getNumStmt() = 0 and
         not ifstmt.hasElse()
 
-#. Click **Run**.
+#. Re-run the query.
 
    There are now fewer results because ``if`` statements with an ``else`` branch are no longer reported.
 
-➤ `See this in the query console <https://lgtm.com/query/1899933116489579248/>`__
+Saving your quick query for future development
```

Comment:
If we can get away with specifying the latest version of the library packs, then maybe this could be turned into a reusable snippet (with a placeholder <lang> instead of cpp as appropriate)?

Reply:
Many thanks for the review.

🤦🏻‍♀️ I copied this here when I started updating the procedure, and then realized that people could just move/copy the `Quick Query` folder created by the extension to a permanent location, as described on line 79.

I meant to delete this section from line 148 to 164 before committing the changes. Unless you think this would be useful, I suggest that we delete this.

Reply:
🔥  (please delete away!)